<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindAxes</class>
 <widget class="QDialog" name="FindAxes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="moveXTo0">
       <property name="text">
        <string>move x to 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="moveXToN">
       <property name="text">
        <string>move x to 102</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="showX">
       <property name="text">
        <string>show x</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="moveYTo0">
       <property name="text">
        <string>move y to 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="moveYToN">
       <property name="text">
        <string>move y to 102</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="showY">
       <property name="text">
        <string>show y</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="moveZTo0">
       <property name="text">
        <string>move z to 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="moveZToN">
       <property name="text">
        <string>move z to 102</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="showZ">
       <property name="text">
        <string>show z</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="swapXY">
       <property name="text">
        <string>swap x y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="swapXZ">
       <property name="text">
        <string>swap x z</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="swapYZ">
       <property name="text">
        <string>swap y z</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
